docs/reference/vpc-association-policy.md

```diff
@@ -0,0 +1,82 @@
+# VpcAssociationPolicy API Reference
+
+## VpcAssociationPolicy
+
+VpcAssociationPolicy is a Custom Resource Definition (CRD) that can be attached to a Gateway to define the ServiceNetworkVpcAssociation configuration.
+
+### Fields of VpcAssociationPolicy
+
+| Field Name	                                                                                                        | Required  | Description	                                        |
+|--------------------------------------------------------------------------------------------------------------------|-----------|-----------------------------------------------------|
+| `apiVersion` *string*	                                                                                             | yes       | ``application-networking.k8s.aws/v1alpha1`` 	       |
+| `kind` *string*	                                                                                                   | yes       | ``VpcAssociationPolicy``                            |
+| `metadata` [*ObjectMeta*](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#objectmeta-v1-meta) | yes     	 | Kubernetes metadata for the resource.               |
+| `spec` *VpcAssociationPolicySpec*	                                                                                 | yes       | Defines the desired state of VpcAssociationPolicy.	 |
+
+
+
+### Fields of VpcAssociationPolicySpec
+
+Appears on: VpcAssociationPolicy
+
+VpcAssociationPolicySpec defines the desired state of VpcAssociationPolicy.
+
+
+
+| Field Name	        | Type                                                                                          | Required | Description                                                                                                                                                                                                                                                                                                       |
+|--------------------|-----------------------------------------------------------------------------------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `targetRef`        | *[PolicyTargetReference](https://gateway-api.sigs.k8s.io/geps/gep-713/#policy-targetref-api)* | Yes	     | TargetRef points to the kubernetes `Gateway` resource that will have this policy attached. This field is following the guidelines of Kubernetes Gateway API policy attachment.                                                                                                                                    |
```

Comment:
"This field is following the guidelines of Kubernetes Gateway API policy attachment." can be shortened to "Follows the guidelines of Kubernetes Gateway API policy attachment", but do we need this sentence if we already link to it in the hyperlink on PolicyTargetReference, or is this a different guideline? If it is a different guideline, we should hyperlink to it